app/src/main/res/values/strings.xml

```diff
@@ -278,17 +278,17 @@
     <string name="app_mode_dns_low_battery">DNS (battery saver)</string>
```

Comment:
```suggestion
    <string name="app_mode_dns_low_battery">DNS (saves battery)</string>
```

Reply:
here: `battery saver` as in `battery saver` mode. Meant as a noun, not a verb.

Reply:
Can it be "Save battery" (mode), and "(Save battery)"?